assets/xml/overlays/ovl_Arrow_Ice.xml

```diff
@@ -1,11 +1,12 @@
 <Root>
+    <!-- Ice Arrow assets -->
     <File Name="ovl_Arrow_Ice" BaseAddress="0x80922430" RangeStart="0x980" RangeEnd="0x1DA0">
         <Texture Name="sIceArrow1Tex" OutName="ice_tex_1" Format="ia8" Width="32" Height="64" Offset="0x980"/>
         <Texture Name="sIceArrow2Tex" OutName="ice_tex_2" Format="ia8" Width="32" Height="64" Offset="0x1180"/>
         <Array Name="sIceArrowVtx" Count="43" Offset="0x1980">
             <Vtx/>
         </Array>
-        <DList Name="sIceArrowDL" Offset="0x1C30"/>
-        <DList Name="sIceArrowVtxDL" Offset="0x1CE0"/>
+        <DList Name="sMaterialDL" Offset="0x1C30"/>
+        <DList Name="sModelDL" Offset="0x1CE0"/>
```

Comment:
I personally don't like dropping the overlay's name from the variable's names.
Also, this kind of naming doesn't follow our naming scheme for extracted assets.

Reply:
Since this asset is identical to OoT, I would really strongly prefer to use the same name for it as OoT, which is what I used. Is the problem that it isn't something like `sIceArrowMaterialDL` and `sIceArrowModelDL`?

Reply:
Yeah, that naming scheme is preferred.
The problem with dropping the overlay name from the variable is it could cause name collisions with other variables since this can be a common name. OoT works around that problem by marking assets extracted from overlays as `static` which we don't do, mainly because we don't think marking extracted assets as static makes much sense

Reply:
Ah, got it. In this instance, I'll relent and update the names. But I still really strongly believe that assets (or, at the very least, DLists that get used in an overlay) that are the same between the two games should be named the same thing. We know from OoT3D/MM3D file names that that's what Nintendo/Grezzo did (check OoT3D's zelda_dog.zar vs. MM3D's zelda2_dog.gar.lzs for example), and even if they *didn't* do that, it just makes good sense to do it ourselves. But this PR is neither the time or place to debate this sort of thing, so I'll switch to `gIceArrowMaterialDL` and `gIceArrowModelDL`.